screens/Map.js

```diff
@@ -35,6 +36,8 @@ function Map ({ navigation }) {
     const [getInsideBuild, setGetInsideBuild] = React.useState("");
     const [mapPressed, setmapPressed] = React.useState("");
 
+    CurrentBuildingLocation();
```

Comment:
I would not call it here neither. Calling the component on line 79 is a good first step.

Reply:
What do you mean ? I cant call it inside the render

Reply:
I got what you mean